test/test_ble_log_policy/test_main.cpp

```
@@ -0,0 +1,55 @@
+
+static void test_interval_is_millis_wrap_safe()
+{
+    constexpr uint32_t startedAtMs = UINT32_MAX - 10;
+    TEST_ASSERT_FALSE(BleLogPolicy::intervalElapsed(true, startedAtMs, 25, 10));
+    TEST_ASSERT_TRUE(BleLogPolicy::intervalElapsed(true, startedAtMs, 25, 14));
+}
+
+void setUp(void) {}
+void tearDown(void) {}
+
+void setup()
```

Cppcheck (2.21.0)

[style] The function 'setup' is never used.

(unusedFunction)